packages/iov-tendermint-rpc/src/rpcclients/websocketclient.ts

```diff
@@ -131,6 +131,40 @@ class RpcEventProducer implements Producer<SubscriptionEvent> {
   }
 }
 
+class ReconnectingRpcEventProducer implements Producer<SubscriptionEvent> {
+  public stopped: boolean = false;
```

Comment:
this can become private, right?

Reply:
Yes, thanks, I originally had the check outside the class.